brazilian portuguese.ini

```
@@ -0,0 +1,666 @@
+﻿[Header]
+Language = "Português Brasileiro"
+Author = "DoctorM64"
+Author2 = "Felipe"

```

Was DoctorM64 really part of this translation?
I doubt that, you probably want it like this instead:
```suggestion
Author = "Felipe"
Author2 = ""
```